region/norfair/crocomire/Post Crocomire Jump Room.json

```
@@ -1239,6 +1280,22 @@
       ],
       "devNote": "While the high pixel position looks possible, it is fairly unreasonable, as it also requires very precise horizontal positioning and the Mellas are chaotic."
     },
+    {
+      "link": [5, 3],
+      "name": "Use Flash Suit",
+      "requires": [
+        {"useFlashSuit": {}},
+        {"or": [
+          {"shinespark": {"frames": 32, "excessFrames": 6}},
```

```suggestion
          {"shinespark": {"frames": 30, "excessFrames": 6}},
```
Use the slope on the far right.